### ISSUE TITLE
Tome of Blood : Rebirth and Ars Elementia 

### ISSUE THREAD

**Issue by SonRaon** [dphaldes/Tome-of-Blood#8]
Hello, 
I know that I'm writing on the issues page, but I'm not writing to report an issue, more so to ask something.

Can the Heretic Armor turn into the elemental armors from Ars Elementia?

I'm asking it, because if it's not possible I'd like to suggest a compatibility between the two of them since it'd make the Heretic Armor all the more versatile if it could be used as a crafting component for the Ars Elementia armors.

**Comment by SonRaon** [comment]
@mystchonky apologies for the tag, but I'd like your opinion on the idea I proposed.

**Comment by dphaldes** [comment]
What exactly do you mean by compatibility ? Would you like some special behaviour ?

**Comment by SonRaon** [comment]
I meant compatibility as in allowing the Heretic Armor to be used as a crafting component when creating the Ars Elementia Armors similar to how the Arcanist Armor is used in... for lack of a better term _vanilla_ Ars Elementia 

I'm asking that since it'd be a shame if someone created the Heretic Armor, only for them to toss it aside for the Ars Elementia armor benefits later on.

If the Heretic Armor was capable of turning into one of the Ars Elementia armor sets, it'd make it even more versatile due to combining the effects of essentially three mods :
- Ars Nouveau
- Ars Elementia and 
- Blood Magic, which would in return offer players a much more versatile gameplay since they'd have more options on how to personalise the upgrades they do to the Heretic Armor.

Not to mention that it'd be balanced out due to people still having to grind for the Tier 3 Heretic Armor before the Ars Elementia upgrade since even in regular Ars Elementia, you can't upgrade the Arcanist Armor until you upgrade it to tier 3.

That's my two cents on what would be an idea for the compatibility between the two, but the choice of it being a thing or not isn't mine to make since you're the author of the mod, not me. I'm just stating my opinion.

**Comment by dphaldes** [comment]
Tbh I am not that familiar with the Elemental Armor sets. I know that they act somewhat like Tier 4 armors. I will consider the idea, but I would like to ensure that the elemental armor sets don't overshadow other armors. I will try to find a way to do so :). If you have any ideas, feel free to drop them here

**Comment by SonRaon** [comment]
To be fair, although if Heretic Armor could be able to turn into a Tier 4/Ars Elementia armor, although it would overshadow the _vanilla_ Ars Elementia armors, due to the sheer amount of upgrades it can get, in terms of a collective Modpack creation it wouldn't be that significant since it'd essentially be Ars Nouveau's "answer" to the more mod heavy, powerful armors.

To better let you understand about the point I'm trying to make, think about it as an alternate path for those who still want to have the benefits of both Ars Nouveau and blood magic, without having to invest to more _grindy_ armor sets like Dragonsteel Armor from Ice and Fire for example, if we postulate that Ice and Fire is the stronger mod in the Modpack in the example.

In other words, although an Elemental version of Heretic Armor would be strong if you have a Modpack of only Ars Nouveau based mods, it wouldn't be **that** OP in a Modpack environment when there are - apologies for the exaggeration - billions of mods that would surpass it in either base stats or versatility.

I'll let you know about any other ideas I may find.

**Comment by SonRaon** [comment]
@mystchonky hello again and apologies in advance in case my tag had interrupted you from anything you were doing.

I'd like to ask if you had thought about including my request as part of your mod addon for Ars Nouveau.

Additionally, I was kinda wrong about the part where I said that the Ars Elementia armors not overshadowing other armors since it appears that some of the Ars Elementia armors have unique set benefits (along with their elemental focuses ofc) :

The Pyromancer Robes gives you a Spell Strength II effect when on fire alongside the ability to apply Hellblaze to the opponent which is essentially an amalgamation of **_irresistible_** fire damage and anti heal.

The Geomancer Focus and Robes increase your Healing strength from any healing effects by 150% on top of nullifying Knockback, respectively 

The Aquamancer Robes and Armor gives Dolphin's Grace and water breathing

The Aethermancer Robes and Focus give an increase in fall and kinetic damage resistance.

These armors also give:
Resistances to attacks of their specific elements

Mana regen when attacked by attacks corresponding to their element or when other conditions are met.

On the other hand if the Heretic/Living Mage Armor would be able to be upgraded to either of these elemental variants, it'd make these armors to get the additional bonuses from Living Armor upgrades, making them ever so much stronger than before.

This would allow players the opportunity to stick with the Living Mage Armor set for longer in their respective playthroughs by reaping the benefits of an elementally upgraded Living Mage Armor.

**Comment by dphaldes** [comment]
I thought about this for a while, and also discussed it with other addon devs and members of the Ars Nouveau community. I consider Heretic armor to be on the same tier as Elemental ones and having elemental stats on Heretic will overshadow elemental armor sets in the current form (the elemental sets itself will lose all value)
Also we are getting in the scope of power creep and Heretic Armor was never supposed to be the best armor for everything.
This is not the final decision. If Ars Elemental decides/wants to create a way to allows the set bonuses on other sets, I will gladly make modifications to allow for them. But in the current state, it is less than ideal and doesn't fit my vision for the addon.
